tests/test_free_fleet.py

```
@@ -38,6 +39,15 @@ def test_every_model_is_from_the_free_list():
             "נכנס" % m)
 
 
+def test_openrouter_models_must_be_free_suffixed():
+    """‏OpenRouter מגיש גם מודלים בתשלום מאותו endpoint — הסיומת
+    ‏:free היא הגדר, וכל מודל בלעדיה הוא הדרך הרביעית שדולר נכנס."""
```

Ruff (0.16.3)

[error] Contains control characters that can permit obfuscated code

(PLE2502)